gum/meson.build

```
@@ -274,7 +274,18 @@ install_headers(gum_arm_headers, subdir: install_header_subdir + '/arch-arm')
 install_headers(gum_arm64_headers, subdir: install_header_subdir + '/arch-arm64')
 install_headers(gum_mips_headers, subdir: install_header_subdir + '/arch-mips')
 
-gum = library('frida-gum-' + api_version, gum_sources,
+gum_enums = gnome.mkenums(
+  'gum-enum-types',
+  sources: gum_headers,
+  c_template: 'gum-enum-types.c.template',
+  h_template: 'gum-enum-types.h.template',
```

```suggestion
  c_template: 'gumenumtypes.c.template',
  h_template: 'gumenumtypes.h.template',
```

For consistency with the style used in this repo.